src/web/app/components/account-requests-table/account-requests-table.component.ts

```diff
@@ -0,0 +1,120 @@
+import { Component, Input } from '@angular/core';
+import { NgbModalRef } from '@ng-bootstrap/ng-bootstrap';
+import { AccountService } from '../../../services/account.service';
+import { SimpleModalService } from '../../../services/simple-modal.service';
+import { StatusMessageService } from '../../../services/status-message.service';
+import { MessageOutput } from '../../../types/api-output';
+import { ErrorMessageOutput } from '../../error-message-output';
+import { SimpleModalType } from '../simple-modal/simple-modal-type';
+import { collapseAnim } from '../teammates-common/collapse-anim';
+
+/**
+ * Contains details about a the account request to be displayed in the list.
+ */
+export interface AccountRequestData {
+  name: string;
+  email: string;
+  status: string;
+  institute: string;
+  country: string;
+  createdAtText: string;
+  registeredAtText: string;
+  comments: string;
+  registrationLink: string;
+  showLinks: boolean;
+}
```

Comment:
Maybe we can move this out into a separate file so that this one is only concerned with the component. Also, I think we usually call this `XyzRowModel`. You can take a look at the [logs table](https://github.com/TEAMMATES/teammates/blob/11b8b818f3a1e2b810de20907725561b009f9297/src/web/app/components/logs-table/logs-table-model.ts) for an example.